Makefile

```diff
@@ -229,6 +232,12 @@ build/assets/%.o: assets/%.c
 	$(CC) -c $(CFLAGS) $(MIPS_VERSION) $(OPTFLAGS) -o $@ $<
 	$(OBJCOPY) -O binary $@ $@.bin
 
+build/include/tables/dmadata_table.h: build/$(SPEC)
```

Comment:
I'm not sure it makes sense to make that file in `build/include/`, it seems like it could be misleading. I'd say we could just have it in root as `build/dmadata_table.h` like other build files we create specifically from the makefile (or maybe `build/dmadata_table_spec.h` or something along those lines so it's not confused with the real `dmadata_table.h`)